external/fv3fit/fv3fit/keras/adapters.py

```diff
@@ -111,7 +111,14 @@ def _rename_graph_outputs_to_match_output_keys(model: tf.keras.Model) -> tf.kera
 
 
 def _ensure_list_input(model: tf.keras.Model) -> tf.keras.Model:
-    # useful for connecting input layers with names different from the original model
+    """
+    Takes dict input + output model and converts it to take in
+    list inputs but keep original dict outputs. Useful for connecting
+    input layers with names different from the original model.
+    
+    Should NOT be used outside of helping rename_dict_input
```

Comment:
I would consider removing this cautionary statement...it does seem like a useful functionality.